.dockerignore

```
@@ -0,0 +1,5 @@
+.env
+.git
+.venv
+Dockerfile
+.vscode
```

Nice, some extras we might consider here include caching files, e.g.:
```suggestion
.vscode
**/__pycache__
**/.pytest_cache
```

(Note `**/` to mean ignore them even if they're nested in a directory/subdirectory etc., which is different to gitignore which does that automatically!)